cmd/daemon/flux-notification-controller/server.go

```diff
@@ -0,0 +1,15 @@
+package flux_notification_controller
+
+import (
+	"net/http"
+
+	"github.com/gorilla/mux"
```

Comment:
Any reason for introducing `mux` here as a dependency compared to the standard libraries `http.ServeMux`?